portfolio/src/main/webapp/script.js

```diff
@@ -380,23 +382,30 @@ function commentsInit() {
  * Initializes the map.
  */
 function mapInit() {
-  const homeIsolabellaCoords = { lat: 44.9067176, lng: 7.9096066 };
-  const universityCoords = { lat: 45.0623602, lng: 7.6599514 };
-
-  const map = new google.maps.Map(document.getElementById("map-container"), {
-    center: homeIsolabellaCoords,
+  const mapContainer = document.getElementById("map-container");
+  const map = new google.maps.Map(mapContainer, {
+    center: HOME_COORDS,
     zoom: 10
   });
 
-  addLandmark(homeIsolabellaCoords, map, "H", "This is my hometown and where I live now.");
-  addLandmark(universityCoords, map, "U", 
-    "This is the Polytechnic of Turin, where I' m studying computer engineering.");
+  addLandmark({
+    location: HOME_COORDS,
+    map: map,
```

Comment:
the cool thing about object initialization is that you can omit the key if you use a variable as the value and the variable name is the same as the key. So here you can write `{location: HOME_COORDS, map, label: "H", description: "something"}`